tests/client/feedback/test_schemas.py

```diff
@@ -233,6 +236,40 @@ def test_rating_question_strategy(strategy, unified_response):
     assert RatingQuestionUnification(question=question, strategy=strategy)
 
 
+@pytest.mark.parametrize(
+    "strategy, unified_response",
+    [
+        ("mean", [{"value": "yes", "strategy": "mean"}]),
+        ("majority", [{"value": "yes", "strategy": "majority"}]),
+        ("max", [{"value": "no", "strategy": "max"}]),
+        ("min", [{"value": "yes", "strategy": "min"}]),
+    ],
+)
+def test_ranking_question_strategy(strategy, unified_response):
```

Comment:
Should we add a test case for ties/draws? How's that being handled now?

Reply:
I will add a test.